components/frontend/src/source/SourceEntityAttribute.js

```diff
@@ -5,10 +5,12 @@ import { TimeAgoWithDate } from '../widgets/TimeAgoWithDate';
 
 export function SourceEntityAttribute({ entity, entity_attribute }) {
     let cell_contents = entity[entity_attribute.key] ?? "";
-    cell_contents = cell_contents && entity_attribute.type === "datetime" ? <TimeAgoWithDate dateFirst date={cell_contents} /> : cell_contents;
-    cell_contents = cell_contents && entity_attribute.type === "date" ? <TimeAgoWithDate dateFirst noTime date={cell_contents} /> : cell_contents;
-    cell_contents = cell_contents && entity_attribute.type === "integer" ? Math.round(cell_contents) : cell_contents;
-    cell_contents = cell_contents && entity_attribute.type === "status" ? <StatusIcon status={cell_contents} /> : cell_contents;
+    const type = entity_attribute.type ?? "";
```

Comment:
Is `type` not some protected name like in python?

Reply:
Nope, `typeof` is, but not `type`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#keywords.